src/server/views/index.pug

```diff
@@ -95,7 +95,9 @@ block content
     include includes/examples.pug
     include includes/pubProjects.pug
   .social-buttons-holder
-    a(href='https://github.com/netsblox/netsblox', target="_blank" title='GitHub')
+    a(href='https://github.com/netsblox', target="_blank" title='Github')
```

Comment:
Is there a reason why you changed the link to the organization instead of the repo?

Reply:
since we have the one frontend that we are using for all the projects under netsblox linking to the organization would be more appropriate rather than linking just to the server side code